### PR TITLE
delete most of Jenkins stuff

### PR DESCRIPTION
**Pull request by krzyzacy**
this is my most pleasant time of today after fighting with Cloud IAP for the entire day and 🤕 
and @fejta can finally burn his Jenkins shirt so that his kid should not have a chance to see it :-)

TODO: 
1. move bootstrap to a different dir & update all images
2. clean up the jenkins project

check_projects.py is still referenced internally (😞)

/area jenkins
/assign @BenTheElder @fejta @ixdy

### CODE REVIEW

**Comment by BenTheElder** [comment]
> move bootstrap to a different dir & update all images

/shrug this is still legacy junk, might as well leave it there
It's not like the jenkins in the service account and all the log paths are going anywhere

File: jenkins/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Jenkins is deprecated
+# Jenkins is ~~deprecated~~ DEAD
```
Comment (BenTheElder) [inline jenkins/README.md]: 💀

File: jenkins/docker_diff.py [hunk 2]
```diff
@@ -1,150 +0,0 @@
-#!/usr/bin/env python
```
Comment (BenTheElder) [inline jenkins/docker_diff.py]: if anyone is still interested in this checkout https://github.com/GoogleCloudPlatform/container-diff for something more mature

**Comment by BenTheElder** [comment]
need to update the tests / build to remove the jenkins stuff :-)

**Comment by krzyzacy** [comment]
```
I0207 05:59:19.340] Call:  gcloud auth activate-service-account --key-file=/etc/service-account/service-account.json
W0207 05:59:39.886] ERROR: (gcloud.auth.activate-service-account) Failed to activate the given service account. Please ensure provided key file is valid.
E0207 05:59:40.899] Command failed
```
that's odd?

**Comment by BenTheElder** [comment]
hmm it passed on rerun... strange

**Comment by BenTheElder** [comment]
hand triggering `pull-test-infra-bazel-canary` finishes and fails on the broken test code before `pull-test-infra-bazel` even finishes the build phase 🙃

**Comment by krzyzacy** [comment]
I thought I've cleaned up those BUILD files

**Comment by BenTheElder** [comment]
Build time:
> W0207 06:42:25.822] INFO: Elapsed time: 23.930s, Critical Path: 8.09s

Test error:
> I0207 06:42:46.345] testgrid/cmd/config/config_test.go:342:17: jenkinsPath declared and not used

**Comment by BenTheElder** [comment]
We can remove these now :-)
```
Testgrid group maintenance-all-daily does not have a matching jenkins or prow job
Testgrid group maintenance-all-hourly does not have a matching jenkins or prow job
Testgrid group kubernetes-update-jenkins-jobs does not have a matching jenkins or prow job
```

**Comment by ixdy** [comment]
cc @zmerlynn

**Comment by krzyzacy** [comment]
@ixdy good catch.. because of instead of revert I made a copy of the raw content :joy: fixed

**Comment by zmerlynn** [comment]
image

**Pull request merged**